k8s-opensearch/OpenSearch_/tls_secret.yml.tftpl

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
You can also make a kubernetes resource using terraform:
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/secret_v1

Reply:
@chrisghill is there a benefit to managing it that way? I feel doing it this way (adding it to extra objects) has the benefit of not having to configure kube provider, and keeping everything related to this OpenSearch deployment clearly managed by this helm release.

Reply:
No strong preference. All the k8s bundles will already have the `kubernetes` provider initialized (they create a secret and rbac binding using that exact resource I linked). Whatever is easiest.

This pattern may be unique to this bundle since I don't usually see an "extraObjects` field, but it should work seamlessly if we switch to cert-manager certificates, right (just a different template)?

Reply:
correct we'd just use a different template to make the cert-manager Certificate CRDs.